Fixing 2 errors in backend

s2sharpit opened the pull request:
For #659 

fixing importing error:
![image](https://github.com/rohansx/informatician/assets/104740281/70089c14-b7d4-4b43-9502-fb467c428cc7)

fixing category TypeError:
<img width="908" alt="image" src="https://github.com/rohansx/informatician/assets/104740281/31a7c512-8f71-4a0d-ab08-7c26b9415d3f">

s2sharpit commented:
@crocmons, I found some errors in the last PR. I have added screenshots for reference. I'm fixing them through this PR.

crocmons commented:
> @crocmons, I found some errors in the last PR. I have added screenshots for reference. I'm fixing them through this PR.

@s2sharpit It is because the database in the utils folder I just fixed with another pr ...it is not included in this pr ...please do exact same with my previous pr #906

s2sharpit commented:
> > @crocmons, I found some errors in the last PR. I have added screenshots for reference. I'm fixing them through this PR.
> 
> @s2sharpit It is because the database in the utils folder I just fixed with another pr ...it is not included in this pr ...please do exact same with my previous pr #906

@crocmons These errors are not in #906, these errors are in the latest pr raised by you #967.

crocmons commented:
> > > @crocmons, I found some errors in the last PR. I have added screenshots for reference. I'm fixing them through this PR.
> > 
> > 
> > @s2sharpit It is because the database in the utils folder I just fixed with another pr ...it is not included in this pr ...please do exact same with my previous pr #906
> 
> @crocmons These errors are not in #906, these errors are in the latest pr raised by you #967.

yeah, I know ..I just say please follow the folder structure eg..@utils/database.. add the database file exactly the same as I created #906 in this pr because the db file should be same as I created API routes

s2sharpit commented:
> > > > @crocmons, I found some errors in the last PR. I have added screenshots for reference. I'm fixing them through this PR.
> > > 
> > > 
> > > @s2sharpit It is because the database in the utils folder I just fixed with another pr ...it is not included in this pr ...please do exact same with my previous pr #906
> > 
> > 
> > @crocmons These errors are not in #906, these errors are in the latest pr raised by you #967.
> 
> yeah, I know ..I just say please follow the folder structure eg..@utils/database.. add the database file exactly the same as I created #906 in this pr because the db file should be same as I created API routes

@crocmons, The import statement is imported as "@/utils/database" not like "@utils/database". There should be slash after "@".

crocmons reviewed (approved):
Thanks for fixing the issue... @s2sharpit

s2sharpit commented:
> @s2sharpit oops! Is that the problem ...I guess my db file was removed by me in this latest pr😂...

@crocmons, Db file was not removed. Instead, there was a mistake made during the file import process, which I have since rectified. 😊

crocmons commented:
> > @s2sharpit oops! Is that the problem ...I guess my db file was removed by me in this latest pr😂...
> 
> @crocmons, Db file was not removed. Instead, there was a mistake made during the file import process, which I have since rectified in this PR. 😊

Ah, ok ..actually I want to say that I just guessed this at that moment.
